### DISCUSSION TITLE
Delete stale user to use its domain name

### DISCUSSION THREAD

**Discussion by Masynchin** [community/community#23053] [New to GitHub]
<p>We want to use specific name for our organization, but there is already a user with this handle. This user is stale, no any contributions since account creation. Can this user be deleted, so we can use this name?</p>

**Comment by ettaboyle** [comment, marked as answer]
<p>Hi <a class="mention" href="/masynchin">@Masynchin</a> – Welcome to the GitHub Community Forums!</p>
<p>We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.</p>
<p>If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our <a href="https://docs.github.com/articles/github-trademark-policy/">Trademark Policy</a> to determine if you should submit a trademark policy violation report</p>
<p>Thank you!</p>

**Reply by grexe** [reply]
I find this answer and the resulting situation very unsatisfactory and unnecessarily restrictive.
Any user can create a personal and even an organization profile, then abandon it, and use up the name for everybody else.

I have a similar problem with the name sen-labs. I even own the domain sen-labs.org, but cannot adopt the matching organization name here because someone created a completely empty profile, so I had to resort to sen-laboratories, which is much longer and does not match.
Can you please help at least in extreme cases (almost abuse of a common infrastructure) like these?

**Reply by chrisbbreuer** [reply]
Truly wild that GitHub favors handle squatters over the people building here

**Comment by ahmedelmoughazy** [comment]
I agree with you guys, this is very bad, I just faced this issue, the only contribution that a user has, is him creating the account, and now I can't use my legal name, because of an account that was created 2010, and never been used, no repos, no commits, absolutely nothing, this is just frustrating.
